task_13/app.js

```diff
@@ -0,0 +1,46 @@
+var Chef = function () {
+    this.dishes = ['Steak', 'Tea', 'Soup', 'Burger']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887245)
```suggestion
    this.dishes = ["Steak", 'Tea', 'Soup', 'Burger']
```